### PR TITLE
Treatment Branch (#58)

### PR DESCRIPTION
**Pull request by grabear**
# MicrobiomeR 0.2.2

## Functional Changes

* Added support for multiple treatments for some plots
  * `get_heat_tree_plots` now produces a `metacoder::heat_tree_matrix`.
  * `correlation_plot` now produces multiple plots for data with more that 2 treatments.
  * Added the `get_treatment_matrix` function
  * Added the `get_correlation_data` function
* Updated `analysis` vignette to demonstrate more than 2 treatment groups.
* Updated internal data files.
  * Added new metadata file with 3 treatment groups (`nephele_metadata3.txt`).
  * Formatted Treatment Group metadata.
* Updated public datasets.
  * Removed Greengenes datasets.
  * Added Silva data with 3 treatment groups.
* Added the `color-palettes` vignette.

### CODE REVIEW

**Comment by sdhutchins** [comment]
The permanova test is breaking, but I can check it out and see what's up.

**Comment by grabear** [comment]
I changed the data so it has to be called differently now:

image

**Comment by sdhutchins** [comment]
> I changed the data so it has to be called differently now:
> 
> image

Perfect!

File: R/correlation-plot.R [hunk 1]
```diff
@@ -172,40 +149,120 @@ get_correlation_plots <- function(obj, primary_ranks, secondary_ranks = TRUE, pa
...
+      message(glue::glue(crayon::green("Generating a Correlation Plot comparing ", crayon::bgWhite({pr}), " with ", crayon::bgWhite({sr}), ".")))
+      corr[[pr]][[sr]] <- do.call(correlation_plot, c(list(obj = obj, primary_rank = pr, secondary_rank = sr), params))
     }
   }
   return(corr)
 }
 
+#' @title Get Correlation Plot Data
+#' @description Get the correlation plot data comparing all of the treatment groups.
+#' @param obj PARAM_DESCRIPTION
```
Comment (sdhutchins) [inline R/correlation-plot.R]: No param descriptions on this function.

Comment (grabear) [inline R/correlation-plot.R]:
```suggestion
#' @param obj An object to be converted to a metacoder object with \code{\link[MicrobiomeR]{object_handler}}.
```

**Comment by grabear** [comment]
> Those are the only changes I could see. Everything looks great!
> 
> image

WE'RE SooOOOOO close!!!.....

**Pull request merged**